src/System.Threading.Tasks/tests/AggregateExceptionTests.cs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Threading.Tasks.Tests
+{
+    public class AggregateExceptionTests
+    {
+        [Fact]
+        public static void RunAggregateException_Constructor()
+        {
+            AggregateException ex = new AggregateException();
```

Comment:
I was about to say that it's strange to have AggregateException tests here rather than in the contract from which AggregateException is exposed... and then I discovered that AggregateException is in fact exposed from System.Threading.Tasks.  That's odd, but... oh well.